source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -537,7 +538,47 @@ void RtCoreIo::read(const DocumentPtr& doc, RtCoreIo::ReadFilter filter)
     }
 }
 
-void RtCoreIo::write(DocumentPtr& doc, RtCoreIo::WriteFilter filter)
+void RtFileIo::read(const FilePath& documentPath, const FileSearchPath& searchPaths, RtFileIo::ReadFilter filter)
+{
+    try
+    {
+        DocumentPtr document = createDocument();
+        XmlReadOptions readOptions;
+        readOptions.skipConflictingElements = true;
+        readFromXmlFile(document, documentPath, searchPaths, &readOptions);
+
+        CopyOptions copyOptions;
+        copyOptions.skipConflictingElements = true;
+
+        read(document, filter);
+    }
+    catch(Exception&)
+    {
+        return;
```

Comment:
Won't it be strange from a user perspective if nothing happened? Maybe we should at the least return a boolean?

Reply:
I wasn't sure what approach we're taking here. Do we capture exceptions or not. I guess looking at the other code we are still using exceptions vs return codes. I'll remove the try/catch. Thanks for noticing this as I forgot to add a comment.